pages/dkp/konvoy/2.3/choose-infrastructure/gcp/quick-start-gcp/index.md

```diff
@@ -18,303 +18,193 @@ Before starting the Konvoy installation, verify that you have:
-
-    gcloud iam service-accounts create "$USER"
-    gcloud projects add-iam-policy-binding $GCP_PROJECT --member="serviceAccount:$USER@$GCP_PROJECT.iam.gserviceaccount.com" --role=roles/editor
-    gcloud iam service-accounts keys create $GOOGLE_APPLICATION_CREDENTIALS --iam-account="$USER@$GCP_PROJECT.iam.gserviceaccount.com"
-    ```
-
-##  Create a VPC network
-
-1.  Export required environment variables:
+## Configure GCP prerequisites
 
-    ```
-    export GCP_PROJECT=eng-ksphere-platform
-    export CLUSTER_NAME=$USER-gcp-quickstart
-    export GCP_NETWORK_NAME=$CLUSTER_NAME
-    export GCP_REGION=us-west1
-    ```
-
-1.  Create the VPC network:
-
-    ```
-    gcloud compute networks create "${CLUSTER_NAME}" --project="$GCP_PROJECT" --subnet-mode=auto --mtu=1460 --bgp-routing-mode=regional
-
-    gcloud compute firewall-rules create "${CLUSTER_NAME}-allow-ssh" --project="$GCP_PROJECT" --network="projects/$GCP_PROJECT/global/networks/${CLUSTER_NAME}" --description=Allows\ TCP\ connections\ from\ any\ source\ to\ any\ instance\ on\ the\ network\ using\ port\ 22. --direction=INGRESS --priority=65534 --source-ranges=0.0.0.0/0 --action=ALLOW --rules=tcp:22
-    ```
-
-### Setup Cloud NAT
-
-Kubernetes nodes, to communicate with the control plane, pull container images from registried (e.g. gcr.io or dockerhub) need to have NAT access or a public ip. By default, the provider creates Machines without a public IP.
-
-1.  Create the router:
-
-    ```
-    gcloud compute routers create "${CLUSTER_NAME}-router" --project="${GCP_PROJECT}" --region="${GCP_REGION}" --network="${GCP_NETWORK_NAME}"
-
-    gcloud compute routers nats create "${CLUSTER_NAME}-nat" --project="${GCP_PROJECT}" --router-region="${GCP_REGION}" --router="${CLUSTER_NAME}-router" --nat-all-subnet-ip-ranges --auto-allocate-nat-external-ips
-    ```
-
-## Build an image
-
-Google Cloud Platform does not publish images. You must first build the image.
-
-<p class="message--note"><strong>Note:</strong> You may also skip this step and reuse one that already exists by exporting it.</p> ????
-
-```
-export IMAGE_NAME=cluster-api-ubuntu-2004-v1-21-10-1651240802
-```
-
-1.  Export required environment variables:
-
-    ```
-    export GCP_PROJECT_ID=$GCP_PROJECT
+1.  Log in to Azure:
+<!NEED CONFIRMATION OF STEPS PRE-RELEASE>
+    ```bash
+    gcp login
     ```
 
-2.  Clone the `image-builder` project:
+    ```sh
+    [
+      {
+        "cloudName": "GCPCloud",
+        "homeTenantId": "a1234567-b132-1234-1a11-1234a5678b90",
+        "id": "b1234567-abcd-11a1-a0a0-1234a5678b90",
+        "isDefault": true,
+        "managedByTenants": [],
+        "name": "Mesosphere Developer Subscription",
+        "state": "Enabled",
+        "tenantId": "a1234567-b132-1234-1a11-1234a5678b90",
+        "user": {
+          "name": "user@gcpmesosphere.onmicrosoft.com",
+          "type": "user"
+        }
+      }
+    ]
+    ```
```

Comment:
This doesn't look right to me, this is Azure output, not google...

Reply:
Yep, just missed that one.  THank you!